migration/sql-files/051-categories.sql

```diff
@@ -0,0 +1,22 @@
+CREATE TABLE categories (
+	created_at timestamp with time zone,
+	updated_at timestamp with time zone,
+	deleted_at timestamp with time zone,
+	id uuid primary key DEFAULT uuid_generate_v4() NOT NULL,
+	name text NOT NULL
+);
+
+CREATE INDEX categories_id_index ON categories (id);
+CREATE UNIQUE INDEX categories_name_idx ON categories (name) WHERE deleted_at IS NULL;
+
+CREATE TABLE workitemtype_categories (
+	created_at timestamp with time zone,
+	updated_at timestamp with time zone,
+	deleted_at timestamp with time zone,
+	category_id uuid NOT NULL,
+	id uuid primary key DEFAULT uuid_generate_v4() NOT NULL,
+	workitemtype_id uuid NOT NULL,
+	UNIQUE (category_id, workitemtype_id)
```

Comment:
@kwk thanks for the explaination. I fixed it. I will reply with exact commit number in some time.